pinot-tools/src/test/java/org/apache/pinot/tools/admin/PinotAdministratorTest.java

```diff
@@ -0,0 +1,170 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pinot.tools.admin;
+
+import java.io.StringWriter;
+import java.util.Map;
+import java.util.TreeMap;
+import org.apache.pinot.common.Versions;
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.assertEquals;
+
+
+public class PinotAdministratorTest {
+
+  @Test
+  public void testHelpParameterShouldOutputAllPossibleParametersAndTheirDescriptions() {
+    // given
+    StringWriter out = new StringWriter();
+    PinotAdministrator pinotAdministrator = new PinotAdministrator(out, new Versions());
+
+    // when
+    pinotAdministrator.execute(new String[]{"AddSchema", "--help"});
+
+    // then
+    assertEquals(out.toString(), "Usage: <main class> AddSchema [-hV] [-exec] [-authToken=<_authToken>]\n"
```

Comment:
this assertEquals is going to be hard to maintain if we add additional commands. was wondering how other project use picocli handles unit-test? do we have a util to generate these expected results?